### PR TITLE
Overlay networks tutorial

### LINKED ISSUE #686: Network tutorial

### PR DESCRIPTION
**Pull request by everett-toews**
Closes #686

### CODE REVIEW

File: _tutorials/2016-03-01-overlay-networks.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+
+[Create and connect to a cluster]({{ site.baseurl }}/docs/tutorials/create-connect-cluster/)
+
+### Overlay networks
+
+An overlay network provides isolation for containers to communicate across all of the segments in your cluster on Carina. This adds a new layer of security to your application deployments and makes it much easier for your containers to communicate with one another.
+
+Among the many benefits are a few key features:
+
+* A secured and isolated environment for the containers in a network
```
Comment (mendeni) [inline _tutorials/2016-03-01-overlay-networks.md]: I think we need to be careful about over-stating that overlay networks are "secure".
They are encapsulated, but not encrypted in our env (i.e. not running over IsolatedNet).

Comment (everett-toews) [inline _tutorials/2016-03-01-overlay-networks.md]: @mendeni How about?

* An isolated network for containers that ensures only the services of your choice are exposed outside of your system

Comment (mendeni) [inline _tutorials/2016-03-01-overlay-networks.md]: :+1:

File: _tutorials/2015-10-20-servicenet.md [hunk 2]
```diff
@@ -14,6 +14,8 @@ topics:
 
 This tutorial describes using the internal network ServiceNet with Carina so that containers will only communicate only by using the internal network.
 
+**Note**: We strongly recommend using an overlay network anytime you need two or more containers to communicate. To do so see [Overlay Networks](/docs/tutorials/overlay-networks/).
```
Comment (kmsholcomb) [inline _tutorials/2015-10-20-servicenet.md]: ... to communicate. For instructions, see [Use overlay networks in Carina](/docs/tutorials/overlay-networks/).

(A slight edit, plus a suggested revision to the article name.)

File: _posts/2016-02-17-overlay-networks.md [hunk 3]
```diff
@@ -13,6 +13,8 @@ categories:
 authorIsRacker: true
 ---
 
+**Note**: The permanent tutorial on overlays networks can be found at [Overlay Networks](/docs/tutorials/overlay-networks/).
```
Comment (kmsholcomb) [inline _posts/2016-02-17-overlay-networks.md]: I guess I'd simplify this a bit, as follows:

**Note:** For tutorial information, see [Use overlay networks in Carina](/docs/tutorials/overlay-networks/).

(Note that I suggest a revised article name.)

**Comment by kmsholcomb** [comment]
Hi @everett-toews. I left a couple comments in the files, but my edits of the tutorial are in the attached Word doc. I suggested a slight reorganization of some sections, so I also am attaching a separate file that shows the suggested reorg (mostly just by showing the headings in the new order I suggest). Please let me know if you have any questions or any feedback for me. Thanks!

[overlay_networks_tutorial_editsKH.docx](https://github.com/getcarina/getcarina.com/files/153615/overlay_networks_tutorial_editsKH.docx)
[mockup_reorg_overlay_networks_tutorial_KH.docx](https://github.com/getcarina/getcarina.com/files/153616/mockup_reorg_overlay_networks_tutorial_KH.docx)

**Pull request merged**